scout/constants/file_types.py

```diff
@@ -1,121 +1,70 @@
-    },
-    "vcf_sv": {
-        "category": "sv",
-        "variant_type": "clinical",
-        "load_priority": 45,
-    },
-    "vcf_sv_mt": {
-        "category": "sv",
-        "variant_type": "clinical",
-        "load_priority": 40,
-    },
-    "vcf_sv_research": {
-        "category": "sv",
-        "variant_type": "research",
-        "load_priority": 145,
-    },
-    "vcf_sv_research_mt": {
-        "category": "sv",
-        "variant_type": "research",
-        "load_priority": 140,
-    },
-    "vcf_str": {
-        "category": "str",
-        "variant_type": "clinical",
-        "load_priority": 50,
-    },
-}
+# Variants in Scout will be loadedfor a case  in the same order as these ordered dictionaries
+ORDERED_FILE_TYPE_MAP = OrderedDict(
+    [
+        ("vcf_cancer", {"category": "cancer", "variant_type": "clinical"}),
+        ("vcf_cancer_sv", {"category": "cancer_sv", "variant_type": "clinical"}),
+        ("vcf_snv_mt", {"category": "snv", "variant_type": "clinical"}),
+        ("vcf_snv", {"category": "snv", "variant_type": "clinical"}),
+        ("vcf_sv_mt", {"category": "sv", "variant_type": "clinical"}),
+        ("vcf_sv", {"category": "sv", "variant_type": "clinical"}),
+        ("vcf_str", {"category": "str", "variant_type": "clinical"}),
+        ("vcf_mei", {"category": "mei", "variant_type": "clinical"}),
+        ("vcf_fusion", {"category": "fusion", "variant_type": "clinical"}),
+        ("vcf_cancer_research", {"category": "cancer", "variant_type": "research"}),
+        ("vcf_cancer_sv_research", {"category": "cancer_sv", "variant_type": "research"}),
+        ("vcf_snv_research_mt", {"category": "snv", "variant_type": "research"}),
+        ("vcf_snv_research", {"category": "snv", "variant_type": "research"}),
+        ("vcf_sv_research_mt", {"category": "sv", "variant_type": "research"}),
+        ("vcf_sv_research", {"category": "sv", "variant_type": "research"}),
+        ("vcf_mei_research", {"category": "mei", "variant_type": "research"}),
+        ("vcf_fusion_research", {"category": "fusion", "variant_type": "research"}),
+    ]
+)
 
-OMICS_FILE_TYPE_MAP = {
-    "fraser": {
-        "format": "tsv",
-        "analysis_type": "wts",
-        "category": "outlier",
-        "sub_category": "splicing",
-        "variant_type": "clinical",
-    },
-    "outrider": {
-        "format": "tsv",
-        "analysis_type": "wts",
-        "category": "outlier",
-        "sub_category": "expression",
-        "variant_type": "clinical",
-    },
-    "fraser_research": {
-        "format": "tsv",
-        "analysis_type": "wts",
-        "category": "outlier",
-        "sub_category": "splicing",
-        "variant_type": "research",
-    },
-    "outrider_research": {
-        "format": "tsv",
-        "analysis_type": "wts",
-        "category": "outlier",
-        "sub_category": "expression",
-        "variant_type": "research",
-    },
-}
+
+ORDERED_OMICS_FILE_TYPE_MAP = OrderedDict(
```

Comment:
Ordering also these.. You never know!

Reply:
Not really needed rn but sure, that might happen in the future!